### PR TITLE
cindy

### PR DESCRIPTION
**Pull request by mizzrawr**
pls check my awesome file

### CODE REVIEW

File: cindy/index.js [hunk 1]
```diff
@@ -1 +1,56 @@
...
+
+// 7. Subtract mangoes from apples and check whether this result is less than oranges divided by mangoes.
+let apploes = apples - mangoes; //the result is 15
+console.log(apploes < oranges / mangoes ? "true" : "false"); //the result is 6, so it's false
+
+//8. Check whether mangoes, apples and oranges are equal.
+console.log((mangoes == apples) == oranges ? "true" : "false");
+
+//9. Check whether the remainder of apples divided by mangoes and the remainder of oranges divided by mangoes are equal.
+let value1 = apples / mangoes;
```
Comment (ciamac-da) [inline cindy/index.js:45]: The solution for 9th exercise is:
console.log((apples % mangoes) == (oranges % mangoes));

**Review by ciamac-da — CHANGES_REQUESTED** [review]
Hey Cindy:) Just try to read my comment about 9th exercise and fix the code. Thanks!

File: cindy/index.js (hunk 1 above)
Comment (mizzrawr) [reply cindy/index.js:45]: Done it! :D Thanks!

Comment (ciamac-da) [reply cindy/index.js:45]: Gracias:)

**Review by ciamac-da — APPROVED** [review]
Gracias:) 👍🏻

**Pull request merged**